apache/zeppelin
# [ZEPPELIN-235] jscs in zeppelin-web

### What is this PR for?
Reopening the PR https://github.com/apache/zeppelin/pull/229 for jscs in zeppelin-web, using following  in .jscsrc

```
{
  "preset": "google",
  "maximumLineLength": {
  "value": 120,
  "allExcept": [ "comments", "regex"]
  }
}
```


### What type of PR is it?
[Improvement]


### What is the Jira issue?
* [ZEPPELIN-235](https://issues.apache.org/jira/browse/ZEPPELIN-235)

### How should this be tested?
Try changing the indentation in any javascript file, and see an error being shown in the terminal on compile/build



### Questions:
* Does the licenses files need update? n/a
* Is there breaking changes for older versions? n/a
* Does this needs documentation? n/a
